app/views/teaching_events/index/_filter.html.erb

```diff
@@ -19,7 +19,6 @@
   <div class="teaching-events__filter--group">
     <%= f.govuk_check_boxes_fieldset(:type, legend: { text: "Event type", tag: nil }) do %>
       <%= f.govuk_check_box :type, "git", label: { text: "DfE Get Into Teaching" } %>
-      <%= f.govuk_check_box :type, "onlineqa", label: { text: "DfE Online Q&A" } %>
```

Comment:
We do use the event type filter to link back to (eg from the About GIT events page - linking back to training provider events only)

I also think if we didn't have this, we'd potentially make it harder to find GIT events, as they are far outnumbered by training provider events - something I'd be really loathe to do

Finally - we may possibly bring back online Q&A events as a separate type in the Spring or Summer team so for now, I'd rather not make any code changes!

Reply:
Yes it is a good question. Worth a ticket on the backlog to investigate?

Reply:
I've already got a ticket about back end changes if online Q&A doesn't come back - I'll add a note to that rather than raise a new ticket

But not something I think we should consider in the meantime - not a priority!